PWGJE/EMCALJetTasks/UserTasks/AliJEQnVectorHandler.cxx

```diff
@@ -328,11 +328,13 @@ bool AliJEQnVectorHandler::LoadOADBCalibrations() {
     if (pathToFileCMVFNS.IsNull()) {
       cout<<"loading local file "<<pathToFileLocal.Data()<<endl;
       fOADBFile = TFile::Open(pathToFileLocal.Data());
-    }
+    } */
```

Comment:
Do you want to keep all of this commented out? It seems that it would only support local files now?

Reply:
Yes. The intention is that it only uses local files, which are downloaded during the addtask.

Reply:
Okay - I don't quite see how this works, but I trust that you've implemented it correctly